.github/workflows/docs.yml

```diff
@@ -3,11 +3,13 @@ name: Build docs
 on:
   push:
     branches: [master]
+  pull_request:
   release:
     types: [published]
 
 jobs:
   build-deploy:
+    if: github.event_name == 'push'
```

Comment:
Shouldn't this also include release event ?

Reply:
Yes, I missed this